native_client/python/client.py

```diff
@@ -4,9 +4,13 @@
 from timeit import default_timer as timer
 
 import argparse
+import tempfile
+import subprocess
 import sys
+import os
 import scipy.io.wavfile as wav
-
+import StringIO
```

Comment:
`import StringIO` does not work on anything above Python 2.7, but `import io.StringIO` should work everywhere :).